File: linda-core/src/main/java/it/unibo/coordination/linda/core/InspectableBulkTupleSpace.java
package it.unibo.coordination.linda.core;

public interface InspectableBulkTupleSpace<T extends Tuple, TT extends Template, K, V>
        extends BulkTupleSpace<T, TT, K, V>, InspectableTupleSpace<T, TT, K, V> {

}

File: linda-core/src/main/kotlin/it/unibo/coordination/linda/core/InspectableBulkTupleSpace.kt
package it.unibo.coordination.linda.core

interface InspectableBulkTupleSpace<T : Tuple<T>, TT : Template<T>, K, V, M : Match<T, TT, K, V>>
    : BulkTupleSpace<T, TT, K, V, M>,
        InspectableLindaTupleSpace<T, TT, K, V, M>